### PR TITLE
OlMfpHandler

### PR DESCRIPTION
**Pull request by thiloSchlemmer**
A handler to display the mfp settings on the map.

### CODE REVIEW

File: test/modules/olMap/handler/mfp/OlMfpHandler.test.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+		it('calculates the intermediate azimuth for a quadrangle polygon', () => {
+			setup();
+			const classUnderTest = new OlMfpHandler();
+			const nonUniformQuadrangle = new Polygon([[[0, 10], [10, 9], [10, 0], [0, -2], [0, 10]]]);
+			const squaredQuadrangle = new Polygon([[[0, 10], [10, 9], [10, 0], [0, -1], [0, 10]]]);
+
+			expect(classUnderTest._getAzimuth(nonUniformQuadrangle)).toBeCloseTo(0.048863, 4);
+			expect(classUnderTest._getAzimuth(squaredQuadrangle)).toBeCloseTo(0.0, 5);
+		});
+	});
```
Comment (taulinger) [inline test/modules/olMap/handler/mfp/OlMfpHandler.test.js:223]: Should we add also tests for other private methods #_getOptimalScale(), #_getPageLabel(), etc?

**Review by taulinger — COMMENTED** [review]
Nicely done!

File: test/modules/olMap/handler/mfp/OlMfpHandler.test.js (hunk 1 above)
Comment (thiloSchlemmer) [reply test/modules/olMap/handler/mfp/OlMfpHandler.test.js:224]: Yes, i have added tests for `_getOptimalScale()` `_getPageLabel() ` and  `_createMpfBoundary()`.

**Review by taulinger — APPROVED** [review]
Let's go for a merge!

**Pull request merged**